src/OpenTelemetry/Metrics/AggregationType.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="AggregationType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Metrics
+{
+    internal enum AggregationType
+    {
+        /// <summary>
+        /// Invalid.
+        /// </summary>
+        Invalid = -1,
+
+        /// <summary>
+        /// Calculate SUM from incoming delta measurements.
+        /// </summary>
+        LongSumIncomingDelta = 0,
```

Comment:
What does `Incoming` mean in this context?

Reply:
Incoming = Measurement. i.e the one reported by user using the counter API.

For sync counter, the measurement reported by the user is deltas.
For async counter, the measurement reported by the user is cumulative.